client/my-sites/promote-post-i2/components/campaign-item/index.tsx

```diff
@@ -62,32 +64,89 @@ export default function CampaignItem( props: Props ) {
 
 	const totalBudgetLeftString = `($${ formatCents( totalBudgetLeft || 0 ) } ${ __( 'left' ) })`;
 	const budgetString = campaignDays ? `$${ totalBudget } ${ totalBudgetLeftString }` : '-';
+	const budgetStringMobile = campaignDays ? `$${ totalBudget } budget` : null;
 
 	const campaignContainsData = isCampaignFinished( status );
+	const statusBadge = (
+		<Badge type={ getCampaignStatusBadgeColor( status ) }>{ getCampaignStatus( status ) }</Badge>
+	);
+	const openCampaignLink = 'javascript:void(0);'; // TODO: Provide a valid link for opening campaign details
```

Comment:
I would use the final URL for the campaign here. We know that the URL will be [this one](https://github.com/Automattic/wp-calypso/pull/76415/files#diff-dce5f2320b6c0937aaf15c7936cdfa0ff8d208b8e2823b55210b04ddcf4917e8R48): `/advertising/:site?/campaigns/:campaignId`

To create the path, please use the utility `getAdvertisingDashboardPath`, so the final URL will look something like this:
```
getAdvertisingDashboardPath(
    `/${ selectedSiteSlug }/campaigns/${ campaign_id }`
)
```